src/Raven.Studio/typescript/viewmodels/database/indexes/additionalAssemblySyntax.ts

```diff
@@ -17,7 +17,7 @@ class additionalAssemblySyntax extends dialogViewModelBase {
 
     copySample(sampleTitle: string) {
         const sampleText = additionalAssemblySyntax.csharpSamples.find(x => x.title === sampleTitle).text;
-        copyToClipboard.copy(sampleText, "Sample has been copied to clipboard", this.dialogContainer);
+        copyToClipboard.copy(sampleText, "Sample has been copied to clipboard");
```

Comment:
do we need this.dialogContainer after this change?

Reply:
Checked all and it seems that copy to clipboard works as intended